### PR TITLE
Add 2 options to runcontainer.sh

### PR DESCRIPTION
**Pull request by nhosoi**
New options:
- --extra-rpm <RPM>: specifies an additional rpm package to install in the container. Default to none. E.g., --extra-rpm diffutils --extra-rpm sudo
- --extra-skip-tag <TAG>: specifies a tag to skip. Default to none. E.g., --extra-skip-tag "tests::use_selinux_role"

Sample command line using the new options:
```
tox -e container-ansible-core-2.15 -- --erase-old-snapshot --extra-rpm diffutils \
--extra-skip-tag "tests::use_selinux_role" --parallel 4 \
--image-name centos-9 tests/tests_*.yml
```

Duration per parallel count:
```
==> /tmp/logging_tests.out.parallel_1 <==
  container-ansible-core-2.15: commands succeeded
  congratulations :)

real	32m7.484s
user	7m17.919s
sys	5m12.431s

==> /tmp/logging_tests.out.parallel_2 <==
  container-ansible-core-2.15: commands succeeded
  congratulations :)

real	22m1.151s
user	7m52.177s
sys	5m35.914s

==> /tmp/logging_tests.out.parallel_3 <==
  container-ansible-core-2.15: commands succeeded
  congratulations :)

real	21m0.654s
user	8m29.469s
sys	5m39.771s

==> /tmp/logging_tests.out.parallel_4 <==
  container-ansible-core-2.15: commands succeeded
  congratulations :)

real	19m55.639s
user	8m51.909s
sys	5m19.470s

==> /tmp/logging_tests.out.parallel_5 <==
  container-ansible-core-2.15: commands succeeded
  congratulations :)

real	21m2.216s
user	9m35.384s
sys	5m52.188s
```
The tests were executed on my laptop. We could see the improvement up to parallel == 4.

### CODE REVIEW

File: src/tox_lsr/test_scripts/runcontainer.sh [hunk 1]
```diff
@@ -315,8 +318,11 @@ run_playbooks() {
     setup_vault "$test_dir" "${test_pb_base}.yml"
     # shellcheck disable=SC2086
     pushd "$test_dir" > /dev/null
-    ansible-playbook -vv ${CONTAINER_SKIP_TAGS:-} -i "$inv_file" ${vault_args:-} \
-        -e ansible_playbook_filepath="$(type -p ansible-playbook)" "${test_pb[@]}"
+    for pb in ${test_pb[@]}; do
```
Comment (richm) [inline src/tox_lsr/test_scripts/runcontainer.sh]: I don't think we should change the behavior of doing `tox -e container ... -- tests/tests_*.yml` - if someone really wants to run multiple playbooks with the same invocation of `ansible-playbook` then I think we should assume that they know what they are doing.  A user can use `--parallel 1` to run each playbook separately.

Comment (nhosoi) [reply src/tox_lsr/test_scripts/runcontainer.sh]: Does this commit https://github.com/richm/tox-lsr/pull/1/commits/654d23a6c6eb5109451baf49e4f0e87825ad695d satisfy your suggestion?

Comment (richm) [reply src/tox_lsr/test_scripts/runcontainer.sh]: > Does this commit [654d23a](https://github.com/richm/tox-lsr/commit/654d23a6c6eb5109451baf49e4f0e87825ad695d) satisfy your suggestion?

yes - thanks!

**Comment by nhosoi** [comment]
Shall I add `tests::use_selinux_role` tag to the other roles as in this pr for the logging?
https://github.com/linux-system-roles/logging/pull/338
Or we'd better leave it to each role owner?

**Comment by richm** [comment]
> Shall I add `tests::use_selinux_role` tag to the other roles as in this pr for the logging? [linux-system-roles/logging#338](https://github.com/linux-system-roles/logging/pull/338) Or we'd better leave it to each role owner?

I would prefer to use the tag named `tests::uses_selinux` because a test might not use the selinux role, but might use one of the selinux modules or commands directly (like the `selinux` role itself)

You can submit PRs for all of the roles if you like - I don't think individual role owners will mind, in fact I think they will welcome the help.

**Pull request merged**